FirebaseStorageSwift/Tests/Integration/StorageAsyncAwait.swift

```diff
@@ -20,7 +20,7 @@ import XCTest
 
 #if swift(>=5.5)
   @available(iOS 15, *)
-  class StorageAsyncAwait: StorageIntegrationCommon {
+  class StorageAsyncAsync: StorageIntegrationCommon {
```

Comment:
I guess it should stay `StorageAsyncAwait` ?

Reply:
Yep. Good catch. Thanks!